common/constants.ts

```diff
@@ -220,3 +220,72 @@ export enum WAZUH_MODULES_ID{
 
 export const AUTHORIZED_AGENTS = 'authorized-agents';
 export const HEALTH_CHECK = 'health-check';
+
+// Health check
+export const HEALTH_CHECK_REDIRECTION_TIME = 300; //ms
+
+// Kibana settings
+// Default timeTilter set by the app
+export const WAZUH_KIBANA_SETTING_TIME_FILTER = {
+  from: "now-24h",
+  to: 'now'
+};
+export const KIBANA_SETTING_NAME_TIME_FILTER = 'timepicker:timeDefaults';
+
+// Default maxBuckets set by the app
+export const WAZUH_KIBANA_SETTING_MAX_BUCKETS = 200000;
+export const KIBANA_SETTING_NAME_MAX_BUCKETS = 'timelion:max_buckets';
+
+// Default metaFields Kibana setting set by the app
+export const WAZUH_KIBANA_SETTING_METAFIELDS = ['_source', '_index'];
+export const KIBANA_SETTING_NAME_METAFIELDS = 'metaFields';
+
+
+// Logger
+// log severity
+
+type WARNING = 'WARNING';
+type INFO = 'INFO';
+type ERROR = 'ERROR';
+export type UILogLevel = WARNING | INFO | ERROR;
+export const LoggerLevels = {
+  WARNING: 'WARNING',
+  INFO: 'INFO',
+  ERROR: 'ERROR',
+};
+
+type UI = "UI";
+type BUSINESS = "BUSINESS";
+type CRITICAL = "CRITICAL";
+export type UIErrorSeverity = UI | BUSINESS | CRITICAL;
+export const UIErrorSeverities = {
+  UI: "UI",
+  BUSINESS: "BUSINESS",
+  CRITICAL: "CRITICAL",
+};
+
+export type UIError = {
+  message: string;
+  error: any;
+  title?: string;
+};
+
+export type UIErrorLog = {
+  context: string;
+  level: UILogLevel;
+  severity: UIErrorSeverity;
+  display?: boolean;
+  store?: boolean;
+  error: UIError;
+  location: string;
+};
+
+export type ErrorOrchestrator = {
+  loadErrorLog: (uiErrorLog: UIErrorLog) => void;
+};
+
+export const ToastColor = {
+  SUCCESS: 'success',
+  WARNING: 'warning',
+  DANGER: 'danger',
+};
```

Comment:
This file contains the defined constants. Maybe you could move the types to another localtion. Maybe a local types for the service. Same for the constants, these seem locals to the service.

Reply:
Yes, I am going to reorganize this, some constants will be used at a general level in the app, and those that are specific to the service I will move. Thx!